utility

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
```

Comment:
This should be `/usr/bin/env python3` if you want to be explicit but also make sure to use the python that is part of the users' environment.  Sometimes we don't want to use /usr/bin/python3 -- e.g. when working in a virtualenv.